### PR TITLE
[upstream-update] Do not specify OpenFlags since it was removed upstr…

### PR DESCRIPTION
**Pull request by gottesmm**
…eam.

This was removed upstream in https://reviews.llvm.org/D47789 since the only
place this flag was used was in the windows implementation where the behavior
triggered by this could be hidden in the implementation instead of being an
argument. As such, this code doesn't compile on master-next.

Since this has an acceptable default argument given the current stable, we can
just fix this on master and everything will work.

rdar://42862352

### CODE REVIEW

**Comment by gottesmm** [comment]
@swift-ci smoke test and merge

**Review by jrose-apple — CHANGES_REQUESTED** [review]
No, you have to remove it from all callers too.

**Comment by gottesmm** [comment]
@jrose-apple done.

File: lib/FrontendTool/FrontendTool.cpp [hunk 1]
```diff
@@ -322,8 +322,8 @@ static bool atomicallyWritingToTextFile(
 
   bool actionFailed = false;
   std::error_code EC =
-      swift::atomicallyWritingToFile(outputPath, /*binary*/false,
-                                     [&](llvm::raw_pwrite_stream &out) {
```
Comment (jrose-apple) [inline lib/FrontendTool/FrontendTool.cpp]: Indentation got messed up here.

**Review by jrose-apple — APPROVED** [review]
Thanks! Sorry for the poor communication.

File: lib/FrontendTool/FrontendTool.cpp (hunk 1 above)
Comment (gottesmm) [reply lib/FrontendTool/FrontendTool.cpp]: llvm style I think would say that it should be two. That being said. I am fine fixing it in a subsequent commit.

**Comment by gottesmm** [comment]
@swift-ci smoke test and merge

**Comment by gottesmm** [comment]
@swift-ci smoke test and merge

**Comment by gottesmm** [comment]
Or I'll just fix it real quick

**Comment by gottesmm** [comment]
@swift-ci smoke test and merge

**Comment by gottesmm** [comment]
@swift-ci smoke test and merge

**Comment by gottesmm** [comment]
@swift-ci smoke test and merge

**Comment by gottesmm** [comment]
@swift-ci smoke test and merge

**Comment by gottesmm** [comment]
@swift-ci smoke test and merge

**Comment by gottesmm** [comment]
@swift-ci smoke test and merge

File: lib/FrontendTool/FrontendTool.cpp (hunk 1 above)
Comment (jrose-apple) [reply lib/FrontendTool/FrontendTool.cpp]: Heh. I looked for it, actually, and they don't say anything about what to do when wrapping lines. I thought clang-format double-indented wrapped lines, but I might be misremembering.

**Pull request merged**